src/js/components/state/overview/StateOverview.jsx

```diff
@@ -109,15 +111,23 @@ export default class StateOverview extends React.PureComponent {
                         <strong>Note:</strong> All data on this page is based on Primary Place of Performance.
                     </div>
                     <div className="state-section__row">
-                        <div className="state-section__viz totals">
-                            <h3 className="state-overview__heading">
+                        <div className="state-section__viz totals-container">
```

Comment:
Not sure if you are seeing the same thing as me but i am seeing this locally.
<img width="1076" alt="Screen Shot 2020-09-03 at 12 30 48 PM" src="https://user-images.githubusercontent.com/14301977/92142863-7801d800-ede2-11ea-8902-a51edc4a6a6c.png">
And this in prod.
<img width="1081" alt="Screen Shot 2020-09-03 at 12 39 39 PM" src="https://user-images.githubusercontent.com/14301977/92142941-923bb600-ede2-11ea-8261-d52e1a8fa882.png">
I guess design should weigh in on this.

Reply:
Isn't this because the note was added above the award breakdown?

Reply:
@jonhill13 Conferred with design. We decided to move the note to the bottom below the table. Fixed.